design.md

```diff
@@ -0,0 +1,22 @@
+## Learning objectives
+
+Know of the existence of the ```bool``` type and its two values.
+Know about boolean operators and how to build logical expressions with them.
+Know of the boolean operator precedence rules.
+
+## Out of scope
+
+Nothing.
+
+## Concepts
+
+This Concepts Exercise's Concepts are:
+
+booleans: know of the existence of the bool type and its two values; know about boolean operators and how to build logical expressions with them; know of the boolean operator precedence rules.
+
+## Prerequisites
+
+This exercise's prerequisites Concepts are: 
+- Know of the existence of the bool type and its two values.
+- Know about boolean operators and how to build logical expressions with them.
+- Know of the boolean operator precedence rules.basics: know how to define methods
```

Comment:
I don't think these are the prerequisites. Looking at the exercise, I think the one prerequisite you have is `basics`:

```suggestion
This exercise's prerequisites Concepts are: 

- `basics`: know how to define functions and return values.
```

Reply:
You are right! I've removed this in a previous PR and merged into master.